code/game/turfs/simulated.dm

```diff
@@ -46,7 +46,7 @@
 		var/mob/living/carbon/M = A
 		switch(wet)
 			if(TURF_WET_WATER)
-				if(!M.slip(4, 2, null, NO_SLIP_WHEN_WALKING))
+				if(!M.slip(1, 1, null, NO_SLIP_WHEN_WALKING))

```

Comment:
This is a huge difference in stun time, not the minor change you'd made it out to be. I don't know about the other maintainers but I don't wanna merge this unless you make it a smaller nerf. Balance for stuff like this should be done in baby steps.